.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
 
   # Automatically sort python imports
   - repo: https://github.com/pycqa/isort
-    rev: 5.11.4
+    rev: 5.12.0
```

Comment:
Already merged this change to main branch :)

Reply:
Thanks! I'll merge the latest update from the main branch here